pubspec.yaml

```diff
@@ -24,6 +24,7 @@ dependencies:
   flutter_localizations:
     sdk: flutter
   flutter_svg: ^2.0.10+1
+  flutter_svg_provider: ^1.0.7
```

Comment:
Please share your approach to needing this package. From how I see things, we have `flutter_svg` which allows you to load SVGs 

Sample:
Asset: `lib/common/widgets/app_bar/app_bar.dart:38`
Network: `lib/features/home/widgets/sponsors_card.dart:108`

Reply:
Usually find it easier to work with but I have made a work around so that we just maintain using flutter_svg only